ocs_ci/ocs/ui/base_ui.py

```diff
@@ -177,6 +177,19 @@ def check_element_text(self, expected_text, element="*"):
         )
         return len(element_list) > 0
 
+    def get_element_by_xpath(self, xpath=""):
```

Comment:
Should we call the method `get_elements_by_xpath` if the goal is to return a list of elements that match the xpath ?